configFlannel.md

```diff
@@ -115,11 +119,15 @@ FLANNEL_OPTIONS="eth0"
 ```
 
 
-* Enable the flanneld service and reboot.
+* Enable the flanneld service, let systemd know that the local flanneld service needs a default route and the local etcd service to be started before it, and reboot.
 
 
 ```
-# systemctl enable flanneld
+# systemctl enable NetworkManager-wait-online.service
+# systemctl enable flanneld.service
+# mkdir /etc/systemd/system/flanneld.service.wants
+# ln -s /usr/lib/systemd/system/etcd.service /etc/systemd/system/flanneld.service.wants/
```

Comment:
A general "After" (as you note, not a "Wants") in the unit file should accomplish this, yes, since it's an ordering requirement that only matters when both are enabled.  There doesn't seem to be an equivalent way to express this using the filesystem, so manually adding a link to ".wants" avoids having to modify or override the unit file as we package it.